README.md

```diff
@@ -30,12 +30,11 @@ It offers a variety of features as follows:
 
 ## Apache RocketMQ Community
 * [RocketMQ Community Incubator Projects](https://github.com/rocketmq)
-* [RocketMQ Community Projects](https://github.com/apache/incubator-rocketmq-externals)
```

Comment:
The external projects releases at their own paces. Links will be added to rocketmq web site instead.